build-settings/gen-flow-types.ts

```diff
@@ -27,22 +37,15 @@ for (const inFile of files) {
         continue;
     }
 
-    const args = [
-        "flowgen",
-        inFile,
-        "-o",
-        outFile,
-        "--add-flow-header",
-        "--no-inexact",
-    ];
-
     try {
-        execFileSync("yarn", args, {cwd: rootDir});
-        console.log(`✅ wrote: ${outFile}`);
+        const source = fs.readFileSync(path.join(rootDir, inFile), "utf-8");
+        let contents: string = compiler.compileDefinitionString(source, {
```

Comment:
Not sure if it's worth it, but there's also a `compiler.comileDefinitionFile(path, options)` ([source](https://github.com/joarwilk/flowgen/blob/master/src/cli/compiler.ts#L124)).  It would remove one more line here but probably not affect performance.

Reply:
I tried that and it looked like it wasn't working, so I switch to `compileDefinitionString`, but it was actual a different error in the try-catch block.  After I fixed the real error, I decided not to switch back.